csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -23,29 +23,30 @@ module Public {
    * content type, or a return kind.
    */
   class SummaryComponent extends TSummaryComponent {
-    /** Gets a textual representation of this summary component. */
-    string toString() {
-      exists(ContentSet c | this = TContentSummaryComponent(c) and result = c.toString())
-      or
-      exists(ContentSet c | this = TWithoutContentSummaryComponent(c) and result = "without " + c)
-      or
-      exists(ContentSet c | this = TWithContentSummaryComponent(c) and result = "with " + c)
+    /** Gets a textual representation of this component used for flow summaries. */
+    string getAccessStep() {
```

Comment:
`getMadRepresentation`? (with suitable mention of models-as-data in the QL doc.

Reply:
Yes, we can rename it!